File: src/main/java/rwt/fview/SizeLevel.java
package rwt.fview;

/**
 * Delineate the available size levels for block sizes.
 * Created by richard todd on 10/11/2016.
 */
enum SizeLevel {
    BYTES(1), KB(1024), MB(1024*1024);

    private final int multiplier;

    int getMultiplier() { return multiplier; }

    SizeLevel(int m) {
        multiplier = m;
    }
}

File: src/rwt/fview/SizeLevel.kt
package rwt.fview

/**
 * Delineate the available size levels for block sizes.
 * Created by richard todd on 10/11/2016.
 */
internal enum class SizeLevel(val multiplier: Long) {
    BYTES(1), KB(1024), MB(1024*1024)
}